internal/csi-addons/rbd/replication.go

```diff
@@ -82,6 +83,12 @@ type ReplicationServer struct {
 	*corerbd.ControllerServer
 }
 
+type localstruct struct {
```

Comment:
Naming a struct `localstruct` is not very practical, can you give it a better name? On the other hand, it seems to contain the same fields as `replication.GetVolumeReplicationInfoResponse`, you could just use that instead as well.